toolkit/resources/imageconfigs/packagelists/iso-initrd-packages.json

```diff
@@ -61,6 +61,7 @@
         "usbutils",
         "util-linux",
         "vim",
+        "words",
```

Comment:
Shouldn't `calamares` depend on `words` instead?

Reply:
No. The terminal installer needs words.